app/models/spree/conekta_oxxo_payment.rb

```diff
@@ -1,20 +1,21 @@
+# frozen_string_literal: true
+
 module Spree
   class ConektaOxxoPayment < Spree::PaymentSource
     attr_accessor :server, :test_mode, :api_key
 
-    def authorize(money, creditcard, gateway_options)
-
+    def authorize(_money, source, _gateway_options)
       ::Conekta.api_key = api_key
       ::Conekta.api_version = "2.0.0"
 
-      order = Spree::Order.find_by(number: creditcard.number)
+      order = Spree::Order.find_by(number: source.number)
 
       begin
         response = ::Conekta::Order.create(payload(order))
         creditcard.update_attribute :conekta_order_id, response.id
```

Comment:
@jtapia I think there is an undefined variable here, `creditcard` was replaced by `source`.

Reply:
thanks @edman193, fixed here https://github.com/magma-labs/solidus_oxxo_pay/pull/9/commits/050aa4e20cc025e5e23ff85e66146be6cf686f59#diff-10ab729171451fa861198bab233a83ceR15